pep-0263.txt

```diff
@@ -6,259 +6,286 @@ Author: mal@lemburg.com (Marc-André Lemburg),
+changeable on a per-source file basis by using a special comment
+at the top of the file to declare the encoding.
 
-    I propose to make the Python source code encoding both visible and
-    changeable on a per-source file basis by using a special comment
-    at the top of the file to declare the encoding.
+To make Python aware of this encoding declaration a number of
+concept changes are necessary with respect to the handling of
+Python source code data.
 
-    To make Python aware of this encoding declaration a number of
-    concept changes are necessary with respect to the handling of
-    Python source code data.
 
 Defining the Encoding
+=====================
 
-    Python will default to ASCII as standard encoding if no other
-    encoding hints are given.
+Python will default to ASCII as standard encoding if no other
+encoding hints are given.
 
-    To define a source code encoding, a magic comment must
-    be placed into the source files either as first or second
-    line in the file, such as:
+To define a source code encoding, a magic comment must
+be placed into the source files either as first or second
+line in the file, such as::
 
-          # coding=<encoding name>
+    # coding=<encoding name>
 
-    or (using formats recognized by popular editors)
+or (using formats recognized by popular editors)::
 
-          #!/usr/bin/python
-          # -*- coding: <encoding name> -*-
+    #!/usr/bin/python
+    # -*- coding: <encoding name> -*-
 
-    or
+or::
 
-          #!/usr/bin/python
-          # vim: set fileencoding=<encoding name> :
+    #!/usr/bin/python
+    # vim: set fileencoding=<encoding name> ::
 
-    More precisely, the first or second line must match the regular
-    expression "^[ \t\v]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)".
-    The first group of this
-    expression is then interpreted as encoding name. If the encoding
-    is unknown to Python, an error is raised during compilation. There
-    must not be any Python statement on the line that contains the
-    encoding declaration.  If the first line matches the second line
-    is ignored.
+More precisely, the first or second line must match the regular
+expression "``^[ \t\v]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)```".
+The first group of this
+expression is then interpreted as encoding name. If the encoding
+is unknown to Python, an error is raised during compilation. There
+must not be any Python statement on the line that contains the
+encoding declaration.  If the first line matches the second line
+is ignored.
 
-    To aid with platforms such as Windows, which add Unicode BOM marks
-    to the beginning of Unicode files, the UTF-8 signature
-    '\xef\xbb\xbf' will be interpreted as 'utf-8' encoding as well
-    (even if no magic encoding comment is given).
+To aid with platforms such as Windows, which add Unicode BOM marks
+to the beginning of Unicode files, the UTF-8 signature
+'``\xef\xbb\xbf``' will be interpreted as 'utf-8' encoding as well
```

Comment:
Same here. Using both '...' and \`\`...\`\` doesn't make any sense and it clutters the output.

If you want to point out that it's a string, you can use \`\`'...'\`\`. See the devguide and codecs module documentation as examples of this usage.